open-metadata-test/open-metadata-fvt/access-services-fvt/subject-area-fvt/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/fvt/CheckSerializationFVT.java

```diff
@@ -44,11 +44,12 @@ public static void main(String[] args) {
             String url = RunAllFVTOn2Servers.getUrl(args);
             runWith2Servers(url);
         } catch (IOException e1) {
-            System.out.println("Error getting user input");
```

Comment:
similar comments on the other FVTs in the use of log vs stdout -- but only this outer wrapper. I won't mark the others up.

Reply:
closing as it's covered by other conversations